From 07b6503c3eeaee895885424ddc6a2918bf816e5b Mon Sep 17 00:00:00 2001
From: Tamal <tasaha@habana.ai>
Date: Wed, 27 Nov 2024 13:46:54 +0200
Subject: [PATCH] [FSW-1318] fs1: add redundant engine number

This patch adds redundant number for io and
compute die engines.

Change-Id: I78f1255688f1f74438ca6119e731d6ae72ac7b1f
Signed-off-by: Tamal <tasaha@habana.ai>
---

diff --git a/include/zephyr/drivers/misc/common/fs1/hl_platform_defs.h b/include/zephyr/drivers/misc/common/fs1/hl_platform_defs.h
index 2e43338..7b974ee 100644
--- a/include/zephyr/drivers/misc/common/fs1/hl_platform_defs.h
+++ b/include/zephyr/drivers/misc/common/fs1/hl_platform_defs.h
@@ -80,10 +80,14 @@
 
 #define FS1_NUM_VDEC_PER_DIE			18
 #define FS1_NUM_VDEC_PER_HALF_DIE		9
+#define FS1_NUM_REDUN_VDEC_PER_HALF_DIE		1
 #define FS1_NUM_NIC_PER_DIE			13
+#define FS1_NUM_REDUN_NIC_PER_DIE		1
 
 #define FS1_NUM_PDMA_PER_DIE			2
+#define FS1_NUM_REDUN_PDMA_PER_DIE		1
 #define FS1_NUM_EDMA_PER_DIE			5
+#define FS1_NUM_REDUN_EDMA_PER_DIE		2
 #define FS1_NUM_XECORE_PER_DIE			17
 
 
